services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -271,3 +277,37 @@ async def service_restart_containers(
         raise LegacyServiceIsNotSupportedError() from error
 
     return NoContentResponse()
+
+
+@router.patch(
+    "/projects/{project_id}/-/project-networks",
+    summary=(
+        "After a change to the workbench the `project networks` need to be updated, "
+        "this will trigger attaching and detaching of networks to ensure services "
+        "are correctly conencted between each other. For now all services are placed "
+        "on the same default network."
```

Comment:
the director-v2 is a service on its own, with its own logic.
Why not just describe something like:
```Updates the project networks according to the current project's workbench``` or similar?

Reply:
much better.